dask/array/core.py

```diff
@@ -1307,6 +1307,10 @@ def ravel(self):
 
     flatten = ravel
 
+    @wraps(np.choose)
+    def choose(self, choices):
+        return elemwise(variadic_choose, self, *choices)
```

Comment:
Hey @jcrist, yeah I can switch it so that the method calls the function. I don't have a strong preference either way.